packages/ti_abusech/manifest.yml

```
@@ -1,13 +1,13 @@
 name: ti_abusech
 title: AbuseCH
-version: "2.6.0"
+version: "2.6.1"
```

```suggestion
version: "2.7.0"
```
As its an enhancement

---

fixed